Description: Upstream changes introduced in version 1.3.7-1ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 grails (1.3.7-1ubuntu1) lucid; urgency=low
 .
   * Set JAVA_HOME in startGrails based on update-alternatives result.
 .
 The person named in the Author field signed this changelog entry.
Author: Marco Vermeulen <vermeulen.mp@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- grails-1.3.7.orig/bin/startGrails
+++ grails-1.3.7/bin/startGrails
@@ -57,6 +57,8 @@ fi
 
 ARGUMENTS=$@
 
+JAVA_HOME=$(update-alternatives --query java | grep Value | sed 's_^Value: __g' | sed 's_/jre/bin/java__g')
+echo "Using Java at: $JAVA_HOME"
 # Attempt to set JAVA_HOME if it's not already set
 if [ -z "$JAVA_HOME" ]; then
 	
